tools/apidiff/diff-bundles.sh

```diff
@@ -0,0 +1,27 @@
+# big, dirty hack - but a nice time saver to produce an API diff between any builds
+# ./diff-bundles.sh base.zip target.zip
+
+# anything alreaady built is wrong
+make clean
+
+# unzip the base/origin bundle zip
+rm -Rf origin
+unzip -d origin $1
+
+# copy files over (dirty!) the currennt install directories
+cp -r origin/2.1/ ../../_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/lib/mono/2.1/
+cp -r origin/Xamarin.iOS/ ../../_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/lib/mono/Xamarin.iOS/
+cp -r origin/Xamarin.TVOS/ ../../_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/lib/mono/Xamarin.TVOS/
+cp -r origin/Xamarin.WatchOS/ ../../_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/lib/mono/Xamarin.WatchOS/
+
+cp origin/Xam*.dll* ../../_mac-build/Library/Frameworks/Xamarin.Mac.framework/Versions/git/lib/mono/ 
+cp -r origin/4.5/ ../../_mac-build/Library/Frameworks/Xamarin.Mac.framework/Versions/git/lib/mono/4.5/
+cp -r origin/Xamarin.Mac/ ../../_mac-build/Library/Frameworks/Xamarin.Mac.framework/Versions/git/lib/mono/Xamarin.Mac/
+
+# cleanup after ourselves
+rm -Rf origin
+
+# run Makefile using the target/currennt bundle
```

Comment:
```suggestion
# run Makefile using the target/current bundle
```

Reply:
yeah, my laptop 'n' key is broken :(
need to set an appointment to the apple store